.github/workflows/spelling.yml

```diff
@@ -0,0 +1,24 @@
+name: Spell checking
+on:
+  push:
+    branches:
+      - "**"
+    tags-ignore:
+      - "**"
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: '15 * * * *'
+
+jobs:
+  build:
+    name: Spell checking
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2.0.0
+      with:
+        fetch-depth: 5
+    - uses: check-spelling/check-spelling@0.0.16-alpha
```

Comment:
I'm not sure if we want to add such as early-phase Action as a dependency to our project. Furthermore, I do not think that this warrants being its own workflow, since we already have a `lint` workflow.

Ideally, a spell checker for our Go source code would be added through our linters. Currently, we use `golangci-lint` as a linter, and there exists a ["misspell"](https://github.com/client9/misspell) linter that we could enable. Unfortunately, it looks like it only catches misspellings in code comments or strings, but not identifiers.

Reply:
[microsoft/terminal](https://github.com/microsoft/terminal), [microsoft/PowerToys](https://github.com/microsoft/PowerToys), [microsoft/winget-cli](https://github.com/microsoft/winget-cli) to name a few repos that rely upon this action. They all like it except when they realize that their spelling is not great :smile:

Reply:
I'm not attached to how you anchor/use this action. I'm applying it by using a template. You should be able to place it elsewhere as fits your workflow.

If you run into problems, I'm happy to try to address them in the action.